db/migrate/20190805135829_backfill_metric_owners.rb

```diff
@@ -0,0 +1,10 @@
+class BackfillMetricOwners < ActiveRecord::Migration
+  def up
+    say_with_time 'Updating metric owners...' do
+      Metric.reset_column_information
```

Comment:
Yes as the previous migration will be run at the same time as this one, we need to reset the column information of the model otherwise it will not know about the newly created columns